contrib/intel_qat/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -ex
+# Install script for Intel QAT on Kernels 5.6, 5.7 and 5.8
+
+VERSION=1.7.l.4.10.0-00014
+mkdir tmp
+mkdir src
+wget https://01.org/sites/default/files/downloads/qat$VERSION.tar.gz -O tmp/qat$VERSION.tar.gz
+tar xvf tmp/qat$VERSION.tar.gz -C src
+chown -R root:root  src
```

Comment:
Thats a very good question, I assumed it was required because it was using kernel headers.
But thats stupid, because it's run as root and root has access anyway... >.<